cli/build.rs

```diff
@@ -322,8 +318,8 @@ deno_core::extension!(
   customizer = |ext: &mut deno_core::ExtensionBuilder| {
     ext.esm(vec![ExtensionFileSource {
       specifier: "ext:cli/runtime/js/99_main.js",
-      code: ExtensionFileSourceCode::LoadedFromFsDuringSnapshot(
-        std::path::PathBuf::from(deno_runtime::js::PATH_FOR_99_MAIN_JS),
+      code: ExtensionFileSourceCode::LoadAtRuntime(
+        PathBuf::from("../runtime/js/99_main.js"),
```

Comment:
Two things here: a) my opinion is that `ExtensionFileSourceCode` is super helpful and we should keep it because this immediately caught my attention; b) why is this file loaded at runtime and not snapshotted?

Reply:
> why is this file loaded at runtime and not snapshotted?

It's loaded at the 'runtime' of this build script, as it was before. I renamed the variant to match precisely what it does since it's a public API and can be used with extension builders or customizers. A name like `LoadFromFsDuringSnapshot` is counterintuitive because it might trick users into thinking its filtered out or otherwise guarded against when not snapshotting, whereas this is more cautionary to avoid using it outside of build scripts.

(FTR the first PR removed `ExtensionFileSourceCode` because there was no load-at-runtime case. It worked by just bundling everything since it only applied to `build.rs` contexts. I still don't know where the size increase came from 🙁)

Reply:
I don't understand the size increase either but I guess the linker cannot prove the source wasn't used.